src/modules/python/pbs/v1/_pmi_cray.py

```diff
@@ -538,4 +538,26 @@ def _pmi_ramp_up(self, hosts):
                         launch(func, cmd)
                         sleep_time = random.randint(1,10)
                         time.sleep(sleep_time)
-        return True
\ No newline at end of file
+        return True
+
+    def _pmi_power_status(self, hosts):
```

Comment:
Can you add a brief description of the function like the way you are doing it for others.

Reply:
Okay.